dex/calc/fees.go

```diff
@@ -14,12 +14,16 @@ import (
 // number of lots in the order. For the quote asset, maxSwaps is not swapVal /
 // lotSize, so it must be a separate parameter. The chained swap txns will be
 // the standard size as they will spend a previous swap's change output.
+// For account-based assets, inputsSize will be zero, and nfo.SwapSize =
```

Comment:
I'm fine with having custom functions for dealing with account-based asset required funds. Outside of the asset backends, `OrderRouter` is the only user of `RequiredOrderFunds` and `RequiredOrderFundsAlt`, and `OrderRouter` is already account-aware (in this work) so the functions are only used in a context where we know already know what kind of asset we're dealing with. So we should be free to diverge these `calc` functions with little ripple-effect. I'm not 100% certain what you mean by "new backend method". Do you mean to calculate the input size? If so, there's really nothing to do. Since `OrderRouter` already knows it's account-based, it never collects the `asset.FundingCoin` (only returned from `asset.OutputTracker` anyway), which is where the input size comes from.